services/repository-managers/src/main/resources/message/emailTeamMembershipInvitationExtendedTemplate.html

```diff
@@ -0,0 +1,24 @@
+<html style="-webkit-box-sizing: border-box;-moz-box-sizing: border-box;box-sizing: border-box;font-size: 10px;-webkit-tap-highlight-color: rgba(0, 0, 0, 0);">
```

Comment:
consider changing name to teamMembershipInvitationExtenededToEmailTemplate.html.  This makes the file list next to the other variation.

Reply:
Fixed